reflekt-plugin/src/test/resources/io/reflekt/plugin/code/generation/data/functions2_test/generatedCode.txt

```diff
@@ -82,9 +82,7 @@ public object ReflektImpl {
                 setOf("io.reflekt.codegen.test.FirstAnnotationTest") -> {
                     when (signature) {
                         "kotlin.Function0<kotlin.Unit>" -> listOf(::fooN1 as KFunction<T>, ::barN1
-                                as KFunction<T>,
-                                io.reflekt.codegen.test.TestFunctions.Companion::foo as
-                                KFunction<T>)
+                                as KFunction<T>)
```

Comment:
hmm why did it change?

Reply:
Since we added a condition about founding only for top-level functions. Since we got the first stable version we have to define our invariants (e.g. should we fins private\protected instances, should we find companion functions and so on, currently I kept only public not abstract instances for classes and objects and public top-level functions)